blocks-config/popup-builder/class-uagb-popup-builder.php

```diff
@@ -167,7 +167,18 @@ public function spectra_popup_shortcode( $attr ) {
 	 * @since 2.6.0
 	 */
 	public function enqueue_popup_scripts() {
-		$args   = array( 'post_type' => 'spectra-popup' );
+		$args   = array(
+			'post_type'      => 'spectra-popup',
+			'posts_per_page' => -1,
+			'meta_query'     => array( // phpcs:ignore WordPress.DB.SlowDBQuery.slow_db_query_meta_query
+				array(
+					'key'     => 'spectra-popup-enabled',
+					'value'   => true,
+					'compare' => '=',
+					'type'    => 'BOOLEAN', // Adjust the type based on your meta field data type.
```

Comment:
The last comment looks out of place, either we remove it or we add comments to all four KV pairs.

```suggestion
					'key'     => 'spectra-popup-enabled', // The meta key.
					'value'   => true, // The meta value to compare with.
					'compare' => '=', // The comparison type.
					'type'    => 'BOOLEAN', // The meta value type.
```

Reply:
Done